mantidproject/mantid
# New minimizer classes function names to conform to coding standards

The new minimizers added by [#16663](https://github.com/mantidproject/mantid/pull/16663) PR were translated from Fortran code, but still used the Fortran code style to make it easier for authors to read the translated code. As this was no longer necessary, the function names were to be altered to conform to camel case, which this PR addresses.

**To test:**
- Ensure all tests pass
- Ensure that for each of the changed files that all function names have been converted and none have been missed.

Fixes #16706 .

_Does not need to be in the release notes._
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
